src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package Duke;
+
+import java.util.Scanner;
```

Comment:
Good explicit listing of imported classes

Reply:
Thank you for your compliment! :)